### ISSUE TITLE
IGV SHA256 mismatch

### ISSUE THREAD

**Issue by tintin42** [Homebrew/homebrew-science#4170]
Issuing "brew install igv" results in:
```
Error: SHA256 mismatch
Expected: f95368bb5c359b28786e8cf2d9c44240f44ad5bad7cd41ba8ae22c3608213535
Actual: 55dabcce11863e0f182a6d773633fba7d64bea464eff25a409909df43fde5c4e
```
Maybe a silent update with the same version number?
I just changed the hash in igv.rb to the current value and everything seemingly works again.

**Comment by ilovezfs** [comment]
```
[ ]	IGV_2.3.80.app.zip	29-Aug-2016 18:32 	25M	 
[DIR]	IGV_2.3.80.app/	29-Aug-2016 18:32 	- 	 
[ ]	IGV_2.3.80.zip	29-Aug-2016 18:30 	27M	 
[ ]	IGV_2.3.80_jre.exe	05-Aug-2016 20:33 	90M	 
```

**Comment by ilovezfs** [comment]
This is now moot because I've shipped 2.3.81 :)
